src/appshell/view/appmenumodel.cpp

```diff
@@ -250,23 +260,23 @@ MenuItem AppMenuModel::addItem()
 MenuItem AppMenuModel::formatItem()
```

Comment:
Can you just disable the format menu if there is no opened score?

Reply:
this is how it works now